sycl/test-e2e/NewOffloadDriver/cpu.cpp

```diff
@@ -0,0 +1,20 @@
+//==--- cpu.cpp - AOT compilation for cpu devices using opencl-aot --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===---------------------------------------------------------------------===//
+
+// REQUIRES: opencl-aot, cpu
```

Comment:
need `any-device-is-cpu` here too i think

Reply:
Same as above comments. Thanks